src/outputs/parthenon_xdmf.cpp

```diff
@@ -462,7 +512,7 @@ static void BlockCoordRegularRef(std::ofstream &xdmf, int nbtot, int ib, int nx,
       "            %d 0 1\n"
       "            1 1 %d\n"
       "          </DataItem>\n",
-      nx + (nx > 1), ib, nx + (nx > 1));
+      nx + 1, ib, nx + 1);
```

Comment:
Oh shoot. I had to change this to support the moving coordinates machinery in 2d I think... Maybe we add a flag for that so this is `nx + (custom_coords && (nx>1))` or something?

Reply:
I didn't think that the moving coordinates machinery even used this function.

Reply:
Oops you're right. I must have changed this when floundering... Sorry about this and good catch.